compass/ocean/tests/drying_slope/drying_slope.cfg

```diff
@@ -0,0 +1,27 @@
+# Options related to the vertical grid
+[vertical_grid]
+
+# Number of vertical levels
+vert_levels = 10
+
+# The type of vertical coordinate (e.g. z-level, z-star)
+coord_type = sigma
+
+# config options for drying slope test cases
+[drying_slope]
+
+# the number of grid cells in x and y
+nx = 6
+ny = 28
```

Comment:
Is `ny` here necessary since it's computed based on `resolution`? It seems like the calculation in `Default.configure` could be moved to `InitialState.run` and then `ny` can be removed here.

Reply:
@sbrus89, the philosophy is that users could reasonably change this.  This is just the default.  If you hard code it in `run()`, a user can't change it.

Reply:
But you are right.  If it is set internally in the code, it shouldn't also be here.